apps/website-25/src/components/courses/SideBar.tsx

```diff
@@ -21,12 +22,13 @@ const SideBar: React.FC<SideBarProps> = ({
       <div className="sidebar__content flex flex-col gap-6">
         {units.map((unit) => (
           <div
+            key={unit.id}
             className={`sidebar__unit border ${isCurrentUnit(unit) ? 'sidebar__unit--active border-color-primary' : 'border-color-divider'} rounded-radius-md p-4 flex flex-col gap-2`}
           >
             <div className="sidebar__unit-item flex flex-col gap-2">
               <p className="sidebar__title uppercase text-size-xs font-bold">Unit {unit.unitNumber}</p>
-              <a href={`/courses/${courseId}/unit/${unit.unitNumber}`} className="sidebar__description subtitle-sm">{unit.title}</a>
-              <p className="sidebar__description text-sm">{unit.description}</p>
+              <a href={ROUTES.makeCoursePageRoute(courseSlug, undefined, Number(unit.unitNumber)).url} className="sidebar__description subtitle-sm">{unit.title}</a>
```

Comment:
this morning we aligned on URLs to be maintained from a single SOT in Airtable, so with that in mind, we should be able to pass around `{unit.url}` or equivalent here instead of constructing the path ourself.
```suggestion
              <a href={unit.url} className="sidebar__description subtitle-sm">{unit.title}</a>
```

Reply:
i worry maintaining `courseSlug` is not enough, as we would have to duplicate route creation in stuff like emails (e.g., if we want to link to specific units in emails reminders). what are your thoughts?